enhancements/etcd/disaster-recovery-with-ceo.md

```diff
@@ -19,86 +19,200 @@ see-also:
-This script adds a member to the etcd cluster membership. This script assumes the certificates are valid.
- 
-### Replacing a single failed member 
-If some etcd members fail, but you still have a quorum of etcd members, you can use the remaining etcd members and the data that they contain to add more etcd members without etcd or cluster downtime. Just running the remove script, and the add script will readd the member to the cluster.
-
-### Recover an etcd member ([usr-local-bin-etcd-member-recover-sh.yaml](https://github.com/openshift/machine-config-operator/blob/3e10747951dd1c3c7a9f131d4c5af7805f19a164/templates/master/00-master/_base/files/usr-local-bin-etcd-member-recover-sh.yaml))
-This script helps recover from a complete loss of a master host. This includes situations where a majority of master hosts have been lost, leading to etcd quorum loss and the cluster going offline. This procedure assumes that you have at least one healthy master host. It restores etcd data from backup to gain etcd quorum on the active master host. The steps to recover include setting up an etcd-signer to obtain certificates for new master hosts and growing them to full membership.
-
-### Recover from expired certificates ([recover-kubeconfig.sh](https://github.com/openshift/machine-config-operator/blob/3e10747951dd1c3c7a9f131d4c5af7805f19a164/templates/master/00-master/_base/files/usr-local-bin-openshift-kubeconfig-gen.yaml))
-This procedure enables to recover from a situation where the control plane certificates have expired.
-
-
 ## Proposal
+The proposal is to transfer the ownership of the DR scripts from MCO, remove all the deprecated DR scripts, and to 
+simplify other existing scripts for backing up and restoring the cluster state. 
 
-## User Stories [optional]
+With the simplification of the scripts, different disaster recovery scenarios are documented properly to utilize the
+simplified scripts along with other Openshift utility commands to achieve the recovery of the cluster. 
 
 ## Implementation Plan
-
-1. Make changes to the scripts as needed
-2. Modify CEO to provide the information needed for the DR scripts to work.
+###  Transfer the ownership of the DR files from the MCO to CEO
+All etcd-scripts should be removed from MCO, and be generated along with the static-pod-resources as configmaps. The 
+scripts should be copied to /usr/local/bin in the init container for a convenient access to the admin user.
+    
+###  Remove the deprecated DR scripts
+The scripts for adding a new etcd member; removing an etcd member; and recovering an etcd-member are no longer needed,
+and should be deleted from the repository.
+    
+### Backing up the cluster state
+The script for backing up the cluster state should back up static-pod-resources for etcd, kube-apiserver, 
```

Comment:
@hexfusion I don't remember trying to provide a script ot backup static pods.

Reply:
I was under the impression that to restore state that we should not only replace state for etcd but also restore the on disk state to the matching revision. Is this not true?

To be clear we backup the latest rev /etc/kuberbetes/static-pod-resources/${service}-pod-N